docs/design/fabric-v2+/fpc-registration.puml

```diff
@@ -7,266 +7,198 @@
-
-database "Local storage Peer1_Org1" as storage1 order 24
-participant "TLCC_Enclave Peer1_Org1" as TLCC1 order 25 #99FF99
-
-participant "ECC Peer1_Org1" as ECC1 order 30
-
-participant "ERCC Peer1_Org1" as ERCC1 order 50
+participant "ECC Peer1_Org1" as ECC1 order 20
+participant "ERCC Peer1_Org1" as ERCC1 order 40
+participant "ERCC Peer1_Org2" as ERCC2 order 50
 
-participant "ECC Peer1_Org2" as ECC2 order 60
-participant "ERCC Peer1_Org2" as ERCC2 order 62
+entity IAS order 1
 
-participant "ECC Peer1_Org3" as ECC3 order 70
-participant "ERCC Peer1_Org3" as ERCC3 order 72
-
-entity Orderer order 100
-
-note left ERCC1
-  ERCC can be **integrated** in ECC or as **seperate** chaincode
-  Integrated:
-  - ERCC and ECC share endorsement policy (but need different "type": org-based vs. attestation-based)
-  - attestations stored ECC namespace
-  Separate: <- we use this one
-  - flexible endorsement policies (ERCC and ECC can have different)
-  - attestations stored in ERCC namespace
+note over IAS, ERCC2
+Types are shown in the fpc-components diagram.
 end note
 
-note left ERCC1
-  FPC chaincode has three different associated endorsement policies (EP):
-  - lifecycle transactions ("normal fabric": approve chaincode definition (define application transaction EP), upgrade, etc ...)
-  - application transactions
-  - FPC-lifecycle transactions (register, key-gen, key-dist, ...)
-
-  Decision: We use a **separate** ERCC.
-  We realized FPC-liflecycle as application transaction EP of ERCC chaincode;
-  should ideally match lifecycle EP which is 'Channel/Application/LifecycleEndorsement' and by default majority of organizations)
-  Note that with the **integrated** approach we have deal with a mix of applications transactions EP and FPC-lifecycle EP.
+note over ERCC1, ERCC2 #FFaa00
+  It is assumed that:
+    -channel has already been created;
+    -ERCC has been installed -- approved by the participating organizations, see lifecycle;
+    -ERCC contains hard-coded the TLCC_MRenclave and the IAS Root Certificate;
+    -ERCC has an Endorsement Policy that requires signatures from a majority of orgs,
+     for security reasons -- namely to avoid unilateral modifications and preserve
+     integrity. As a result, changes to chaincode enclave info stored in ERCC will
+     be dependent on such policy.
 end note
 
-ref over User1, ERCC1
-  Create channel; each peer joins channel; each peer launches TLCC, which will  "imprint" itself with CH_id (content-addressable channel identifier, hash of gensis block)
-end ref
-|||
-
-ref over User1, ERCC1
-  Install and approve ERCC chaincode. ERCC defines TLCC_MR and IAS_PK in chaincode definition.
-  (alternatively, TLCC_MR and IAS_PK could be in channel configuration; crucial is that they are approved by humans (admins) on sufficiently many organizations)
-  ERCC also reads CH_id from ledger or, if that does not work, also embeds it into chaincode
-
-  see fpc-lifecycle-v2.puml
-end ref
-|||
-
```

Comment:
I think it makes it easier to understand when the enclave registration happens when we keep the reference to other diagrams here. So why removed?